tests/main/interfaces-framebuffer/task.yaml

```diff
@@ -16,7 +16,7 @@ execute: |
 
     if [ ! -e /dev/fb0 ]; then
         echo "Framebuffer not available on /dev/fb0"
-        exit 0
+        exit 1
```

Comment:
<3

Reply:
```
2018-06-19 14:08:47 Error executing google:fedora-28-64:tests/main/interfaces-framebuffer : 
-----
+ echo 'The plug is not connected by default'
The plug is not connected by default
+ MATCH '\- +test-snapd-framebuffer:framebuffer'
+ snap interfaces -i framebuffer
+ '[' '!' -e /dev/fb0 ']'
+ echo 'Framebuffer not available on /dev/fb0'
Framebuffer not available on /dev/fb0
+ exit 1
-----
```
Yet, my Fedora 27 & 28 images taken from https://cloud.fedoraproject.org come up with `/dev/fb0`.

Reply:
The following tests will be skipped:
    - google:debian-9-64:tests/main/interfaces-framebuffer
    - google:fedora-28-64:tests/main/interfaces-framebuffer
    - google:ubuntu-14.04-64:tests/main/interfaces-framebuffer
    - google:ubuntu-16.04-32:tests/main/interfaces-framebuffer
    - google:ubuntu-16.04-64:tests/main/interfaces-framebuffer
    - google:ubuntu-18.04-64:tests/main/interfaces-framebuffer
    - google:ubuntu-core-16-64:tests/main/interfaces-framebuffer